pkg/api/v1/types.go

```diff
@@ -2412,6 +2412,12 @@ type PodSpec struct {
 	// +patchMergeKey=IP
 	// +patchStrategy=merge
 	HostAliases []HostAlias `json:"hostMappings,omitempty" patchStrategy:"merge" patchMergeKey:"IP" protobuf:"bytes,23,rep,name=hostMappings"`
+	// If specified, indicates the pod's priority. "system" is a special keyword
+	// which indicates the highest priority. Any other name must be defined in
+	// Admission Controller config or pod will be rejected.
```

Comment:
I don't think the admission controller name should be in the API - that has nothing to do with the end user's perception of the field.  A custom admission controller can handle this as well as anything else.